### DISCUSSION TITLE
I have done everything right, but can not seem to switch projects

### DISCUSSION THREAD

**Discussion by gjerd** [mfussenegger/nvim-jdtls#304] [General]
Switching projects (for those in the workspace directory; can anyone
tell me which lua function i have to call; what is the procedure for this?

**Comment by gjerd** [comment]
I see that your plugin switch projects automatically. I would request some deeper understanding in the docs (README.md?). Thank you mfussenegger... awesome plugin I must say. Thanks

**Comment by mfussenegger** [comment]
Sorry but can you elaborate? I have no idea what you're trying to do or what doesn't work.

**Reply by gjerd** [reply]
Well I did confuse myself; what I am doing in my IDE is to open the new projects by creating a new embedded neovim-instance and pointing "--data" in the following manner:


        cmd = {
            "java",
            "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n",
            "-Declipse.application=org.eclipse.jdt.ls.core.id1",
            "-Dosgi.bundles.defaultStartLevel=4",
            "-Declipse.product=org.eclipse.jdt.ls.core.product",
            "-Dlog.level=ALL",
            "-Dlog.protocol=true",
            "-XX:+UseStringDeduplication",
            "-Xmn128m",
            "-Xms1024m",
            "-Xmx3G",
            "-XX:+UseG1GC",
            "-javaagent:"..lombokfile,
            "--add-modules=ALL-SYSTEM",
            "--add-opens", "java.base/java.util=ALL-UNNAMED",
            "--add-opens", "java.base/java.lang=ALL-UNNAMED",
            "-jar", spades_jdt_launcher,
            "-configuration", spades_jdt_directory.."/config_linux",
            "-data", spades_workspace_folder.."/"..vim.api.nvim_get_var("spades_current_project")
        },

By switching project I mean loading a new project in my IDE so that autocompletion works for seperate projects too... As a beginner in lua it did not come clear to me how to do that programmatically. I felt that the old version of docs did not explain this thourougly, Sorry if it does that now.
